drake/systems/plants/RigidBodySystem.cpp

```diff
@@ -134,9 +134,11 @@ RigidBodySystem::StateVector<double> RigidBodySystem::dynamics(
           tangent1 /= sqrt(this_normal(1) * this_normal(1) +
                            this_normal(0) * this_normal(0));
         }
-        Vector3d tangent2 = tangent1.cross(this_normal);
+        Vector3d tangent2 = this_normal.cross(tangent1);
```

Comment:
This is right now because a right handed frame x,y,z should have the circular property x=yXz, y=zXx, z=xXy. The bug was an example of a notational error IMO because "tangent1" and "tangent2" are insufficient to convey handedness. But ... creating a frame given one of its axes is such a common operation that it ought to be done in a utility method so it only has to be done once! I'm not sure if Eigen has that operation, but it does have one for creating a [unit orthogonal vector](http://eigen.tuxfamily.org/dox/classEigen_1_1MatrixBase.html#a42fdca0c5854b85d9482853c8a085dc1) to a given vector.